app/core/utils.js

```diff
@@ -196,6 +196,21 @@ utils.getInviteKey = function(location) {
   return '';
 }
 
+utils.getRoles = function(location) {
```

Comment:
@jebeck Good call. I added tests for `getRoles`. While I was at it, I added tests for `getInviteKey`.